interface/patient_file/encounter/forms.php

```diff
@@ -632,10 +638,11 @@ function divtoggle(spanid, divid) {
         echo "<div class='form_header_controls'>";
 
         // If the form is locked, it is no longer editable
-        if ( $esign->isLocked() ) {
-            echo "<a href=# class='css_button_small form-edit-button-locked' id='form-edit-button-".attr($formdir)."-".attr($iter['id'])."'><span>".xlt('Locked')."</span></a>";
-        } else {
-          if (!$aco_spec || acl_check($aco_spec[0], $aco_spec[1], '' , 'write')) {
+        if ($esign->isLocked()) {
+                 echo "<a href=# class='css_button_small form-edit-button-locked' id='form-edit-button-" . attr($formdir) . "-" . attr($iter['id']) . "'><span>" . xlt('Locked') . "</span></a>";
+          } else {
+          if ((!$aco_spec || acl_check($aco_spec[0], $aco_spec[1], '' , 'write') AND $GLOBALS['enable_group_therapy'] == 0)
+              OR (((!$aco_spec || acl_check($aco_spec[0], $aco_spec[1], '' , 'write')) AND $GLOBALS['enable_group_therapy'] AND acl_check("groups","glog",false, 'write')))) {
```

Comment:
@oshriromatrix 
Hi Rod,
you build a separate acl permission for form.
if some one, wants in the future to add acl to a form, he can use your acl form model.